yt_dlp/extractor/truth.py

```diff
@@ -0,0 +1,91 @@
+from .common import InfoExtractor
+
+from ..utils import (
+    clean_html,
+    int_or_none,
+    unified_timestamp,
+    strip_or_none,
+    traverse_obj
+)
+
+
+class TruthIE(InfoExtractor):
+    """Extract videos from posts on Donald Trump's truthsocial.com."""
+
+    _VALID_URL = r'https://truthsocial\.com/@[^/]+/posts/(?P<id>[\d]+)'
+    _TESTS = [
+        {
+            'url': 'https://truthsocial.com/@realDonaldTrump/posts/108779000807761862',
+            'md5': '4a5fb1470c192e493d9efd6f19e514d3',
+            'info_dict': {
+                'id': '108779000807761862',
+                'ext': 'qt',
+                'title': '0d8691160c73d663',
+                'description': '',
```

Comment:
You sure the description is literally an empty string? Lack of description should be denoted by `None`

Reply:
Fixed.